public/js/dashboardCategories.js

```diff
@@ -25,33 +25,63 @@ async function getAllAcceptedCategories() {
   const responseData = await (
     await fetch(`/data/acceptedcategories/organization/${organizationData.id}`)
   ).json();
-  const responseDataJSONArray = await Object.values(responseData);
-  const allAcceptedCategories = responseDataJSONArray.map(function (organizationCategory) {
-    const categoryName = organizationCategory.category._path.segments[1];
+  const responseDataJSONArray = await Object.entries(responseData);
+  const allAcceptedCategories = responseDataJSONArray.map(function (categoryInfo) {
+    const categoryKey = categoryInfo[0];
+    const categoryValues = categoryInfo[1];
+    const categoryName = categoryValues.category._path.segments[1];
     return {
       name: categoryName.replace(/-/g, ' '),
-      ID: `${categoryName}-category-card`,
-      instructions: organizationCategory.instructions,
-      quality: organizationCategory.qualityGuidelines,
+      ID: categoryKey,
+      instructions: categoryValues.instructions,
+      quality: categoryValues.qualityGuidelines,
     };
   });
   return allAcceptedCategories;
 }
 
+/**
+ * Converts the values of specified input elements to an array.
+ * @param {string} type Name of container to retrieve instructions from.
+ * @return {Array<string>} Values from the specified input container.
+ */
+async function inputsToArray(type) {
+  const inputArray = [];
+  const instructionInputs = await document.querySelectorAll(`.${type}-container > input`);
+  Array.prototype.forEach.call(instructionInputs, function (instruction) {
+    inputArray.push(instruction.value);
+  });
```

Comment:
```instructionInputs.forEach```?

Reply:
Since I'm iterating over `instructionInputs` which is a `NodeList`, the Mozilla docs suggested the current iteration method for compatibility across all browsers here: https://developer.mozilla.org/en-US/docs/Web/API/NodeList.